module/userProjects/helper.js

```diff
@@ -174,20 +184,60 @@ module.exports = class UserProjectsHelper {
 					}
 				}
 
-				if (userProject[0].lastDownloadedAt.toISOString() !== lastDownloadedAt) {
+				if (!invokedViaUpdateApi && userProject[0].lastDownloadedAt.toISOString() !== lastDownloadedAt) {
 					throw {
 						status: HTTP_STATUS_CODE.bad_request.status,
 						message: CONSTANTS.apiResponses.USER_ALREADY_SYNC,
 					}
 				}
 
-				if (userProject[0].status == CONSTANTS.common.SUBMITTED_STATUS) {
+				if (!invokedViaUpdateApi && userProject[0].status == CONSTANTS.common.SUBMITTED_STATUS) {
 					throw {
 						status: HTTP_STATUS_CODE.bad_request.status,
 						message: CONSTANTS.apiResponses.FAILED_TO_SYNC_PROJECT_ALREADY_SUBMITTED,
 					}
 				}
 
+				let currentReflection = userProject[0].reflection || {}
+
+				if (data.reflectionStatus == 'started') {
```

Comment:
@borkarsaish65 we might have constant defined for started and completed, please use them

Reply:
done